x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/components/docker_instructions_step.tsx

```diff
@@ -43,6 +47,19 @@ export const DockerInstructionsStep: React.FC<DockerInstructionsStepProps> = ({
     }
   }, [isWaitingForConnector]);
 
+  const configYamlContent = `connectors:
+  - 
+    connector_id: "${connectorId}"
+    service_type: "${serviceType}"
+    api_key: "${apiKeyData?.encoded ?? ''}"
```

Comment:
@jedrazb this is currently duplicating the ES API key, is that something that should be handled in a nicer way?

Reply:
I opted to reuse getConnectorTemplate in https://github.com/elastic/kibana/pull/197199/commits/c4319f8dbda536c751f7170e20708c555761af0a which didn't need to be hardcoded